ifrs17-template/Test/TestData.ipynb

```diff
@@ -0,0 +1,366 @@
+{
+    "metadata": {
+        "authors": [],
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "code",
+            "source": [
+                "#!import \"../../ifrs17/CalculationEngine\""
```

Comment:
You can't import CE in this way.

Reply:
Later on when the file will be moved in the CE, it can be done in this way.
Do you think it is fine to keep it like this, or change to the one from the Constants folder?

Reply:
This code does not work in the real set up. It works only on your local project where you upload everything in one project. I don't know exactly what you need here... but if it is not clear how to proceed, you should work with 3 projects instead.

Reply:
This code does not work in the real set up. It works only on your local project where you upload everything in one project. I don't know exactly what you need here... but if it is not clear how to proceed, you should work with 3 projects instead.

Reply:
I want the "CalculationEngine" notebook to be imported from the CE. 
How would you import it here?